pkg/kubelet/lifecycle/predicate_test.go

```diff
@@ -147,8 +147,21 @@ func makeAllocatableResources(milliCPU, memory, pods, extendedA, storage, hugePa
 func newResourcePod(usage ...schedulerframework.Resource) *v1.Pod {
```

Comment:
can you instead convert this to `newResourcePod(containerResources []v1.ResourceList)`?

Reply:
Done